File: LV2/2.2/akcija/akcija.cpp
#include <algorithm>
#include <cstdio>
using namespace std;

bool cmp(int a, int b) { return a > b; }

int main() {
    int n = 0;
    int price[1000010] = {0};
    scanf("%d", &n);

    for (int i = 0; i < n; i++) {
        scanf("%d", &price[i]);
    }

    sort(price, price + n, cmp);

    int sum = 0;
    for (int i = 0; i < n; i += 3) {
        sum += price[i] + price[i + 1];
    }
    printf("%d\n", sum);
}

File: LV2/2.2/akcija/akcija.go
package main

import (
	"fmt"
	"sort"
)

func main() {
	n := 0
	price := make([]int, 100010)
	fmt.Scanf("%d", &n)

	for i := 0; i < n; i++ {
		fmt.Scanf("%d", &price[i])
	}

	sort.Sort(sort.Reverse(sort.IntSlice(price[0:n])))

	sum := 0
	for i := 0; i < n; i += 3 {
		sum += price[i] + price[i+1]
	}
	fmt.Println(sum)
}
